### PR TITLE
Add DareData Engineering

### PR DESCRIPTION
**Pull request by ruimgf**
Add Company DareData Engineering

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -135,6 +135,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [BI4ALL](https://www.bi4all.pt/en/) [:rocket:](https://www.linkedin.com/company/bi4all/jobs/) | Consulting services with several analytics solutions. | `Lisboa` `Porto` |
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa` <br> `Porto` `Tomar` <br> `Vila Real` `Viseu` |
 | [Eyenov](https://eyenov.com) | Computer Vision. Machine Learning. Mobile, desktop and embedded. | `Lisboa` `Remote` |
+| [DareData Engineering](https://daredata.engineering/)| Solving Data Science and Data Engineering problems for startups and enterprises. |`Remote`|
```
Comment (marmelo) [inline README.md:138]: Does it make sense to link https://www.daredata.engineering/join-the-team as a :rocket: career's page?
Also, the description is a bit long, can you please try to shorten it?

Comment (marmelo) [inline README.md:138]: Missing spacing.

```suggestion
| [DareData Engineering](https://daredata.engineering/)| Solving Data Science and Data Engineering problems for startups and enterprises. | `Remote` |
```

**Review by marmelo — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -135,7 +135,7 @@ Checking out the company's Tech stack through [Stackshare](https://stackshare.io
 | [BI4ALL](https://www.bi4all.pt/en/) [:rocket:](https://www.linkedin.com/company/bi4all/jobs/) | Consulting services with several analytics solutions. | `Lisboa` `Porto` |
 | [Critical Software](https://www.criticalsoftware.com) [:rocket:](https://www.criticalsoftware.com/careers) | Mission and business-critical applications. | `Coimbra` `Lisboa` <br> `Porto` `Tomar` <br> `Vila Real` `Viseu` |
 | [Eyenov](https://eyenov.com) | Computer Vision. Machine Learning. Mobile, desktop and embedded. | `Lisboa` `Remote` |
-| [DareData Engineering](https://daredata.engineering/)| Solving Data Science and Data Engineering problems for startups and enterprises. |`Remote`|
+| [DareData Engineering](https://www.daredata.engineering/join-the-team)| Data Science, Data Engineering and MLOps Consulting Boutique. | `Remote` |
```
Comment (marmelo) [inline README.md:138]: Sorry, I think I did not express myself correctly.
The company website was OK, I was suggesting adding that link to the carrers page (if you believe it is relevant).

Something like:

```suggestion
| [DareData Engineering](https://daredata.engineering/) [:rocket:](https://www.daredata.engineering/join-the-team) | Data Science, Data Engineering and MLOps Consulting Boutique. | `Remote` |
```

**Review by marmelo — CHANGES_REQUESTED** [review]

File: README.md (hunk 2 above)
Comment (ruimgf) [reply README.md]: Got it :)

**Review by vpereira01 — APPROVED** [review]

**Pull request merged**